{"schemaVersion":{"patch":0,"major":0,"minor":3},"sections":[],"kind":"symbol","abstract":[{"text":"Case represents network error","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["Mac Catalyst"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"network"},{"kind":"text","text":"("},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO","text":"CommunicatorError","kind":"typeIdentifier"},{"kind":"text","text":")"}]}]}],"variants":[{"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"role":"symbol","modules":[{"name":"PasswordAndNfinityFramework"}],"roleHeading":"Case","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO7networkyAC012CommunicatorD00hF0O_tcACmF","title":"NfinityCloudError.network(cause:)","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"network","kind":"identifier"},{"text":"(","kind":"text"},{"text":"cause","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}]},"identifier":{"url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"NfinityCloudError"}],"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"type":"topic","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"kind":"symbol","title":"NfinityCloudError"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)","kind":"symbol","role":"symbol","title":"NfinityCloudError.network(cause:)","abstract":[{"type":"text","text":"Case represents network error"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"network","kind":"identifier"},{"text":"(","kind":"text"},{"text":"cause","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"kind":"text","text":")"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"abstract":[],"role":"collection","title":"PasswordAndNfinityFramework","kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","type":"topic","url":"\/documentation\/passwordandnfinityframework"}}}